src/Steam.package/STApplicationRootComponent.class/instance/storeToUse.st

```diff
@@ -0,0 +1,3 @@
+initialization
+storeToUse
+	^ STStore
```

Comment:
Hmm, this is what we were talking about before. We should return a store object, not a store class. Otherwise we cannot configure it!

Also, it is wrong to return an abstract class. The two correct options would be
- either you mark the method as subclass responsibility and you oblige all users to redefine it
- or you choose a default value (like `STMemoryStore`) that people will override only if they want.

Take into account that a good default value is almost never overridden, because is a good value for most cases. In this case I'd not go with a default value because I don't think there is a good one ^^

Reply:
the use of STStore will raise an error with a message of usage for stores, isn't it enough ?

Reply:
Maybe, but you're kind of forcing it. People do not expect that. Even if in Pharo you can do this, people would expect a more normal pattern, as the ones described above.

I mean, using Pharo specific things in your program is good, if they add really a value. In this case, you can use a more "well-known" pattern (imagine for a java/ruby developer) while still having the same behaviour... so you don't lose anything ^^